docs/proposals/high-availability.md

```diff
@@ -0,0 +1,34 @@
+# High Availability of Daemons in Kubernetes
+This document serves as a proposal for high availability of the master daemons in kubernetes.
+
+## Design Options
+1. Hot Standby Daemons: In this scenario, data and state are shared between the two deamons such that an immediate failure in one daemon causes the the standby deamon to take over exactly where the failed daemon had left off.  This would be an ideal solution for kubernetes, however it poses a series of challenges in the case of controllers where daemon-state is cached locally and not persisted in a transactional way to a storage facility.  As a result, we are **NOT** planning on this approach. 
```

Comment:
Read further, I saw that your "cold standby" is a process that just hangs around waiting to get the lease. So it's not the hybrid. It may be worth mentioning the hybrid as a design alternative you don't plan to use. (I also see that your cold standby is a little warmer than what I described; it's a process with no state yet and that is just trying to grab the lease.)